---
layout: post
title:  "Analysis of Metagenomes"
date:   2013-09-20
excerpt: "A user’s guide to quantitative and comparative analysis of metagenomic datasets."
authors: 'Luo C•, Rodriguez-R LM•, Konstantinidis KT.'
citation: 'Methods in Enzymology 531:525-547. 2013.'
equal_contribution: true
publication: true
tag:
- metagenomics
doi: 10.1016/B978-0-12-407863-5.00023-X
comments: true
---

## Abstract

Metagenomics has revolutionized microbiological studies during the past decade and provided new insights into the diversity, dynamics, and metabolic potential of natural microbial communities. However, metagenomics still represents a field in development, and standardized tools and approaches to handle and compare metagenomes have not been established yet. An important reason accounting for the latter is the continuous changes in the type of sequencing data available, for example, long versus short sequencing reads. Here, we provide a guide to bioinformatic pipelines developed to accomplish the following tasks, focusing primarily on those developed by our team: (i) assemble a metagenomic dataset; (ii) determine the level of sequence coverage obtained and the amount of sequencing required to obtain complete coverage; (iii) identify the taxonomic affiliation of a metagenomic read or assembled contig; and (iv) determine differentially abundant genes, pathways, and species between different datasets. Most of these pipelines do not depend on the type of sequences available or can be easily adjusted to fit different types of sequences, and are freely available (for instance, through our lab Web site: [http://www.enve-omics.gatech.edu/](http://www.enve-omics.gatech.edu/)). The limitations of current approaches, as well as the computational aspects that can be further improved, will also be briefly discussed. The work presented here provides practical guidelines on how to perform metagenomic analysis of microbial communities characterized by varied levels of diversity and establishes approaches to handle the resulting data, independent of the sequencing platform employed.
