---
layout: publication
title:  "Kalamas Genome diversity"
date:   2018-11-05
excerpt: "Quantifying the changes in genetic diversity within sequence-discrete bacterial populations across a spatial and temporal riverine gradient."
authors: 'Meziti A•, Tsementzi D•, Rodriguez-R LM•, Hatt JK, Karayanni H, Kormas KA, Konstantinidis KT.'
citation: 'ISME J 13:767-779. 2018.'
category: Publication
equal_contribution: true
tag:
- genomics
- evolution
- metagenomics
- microbial ecology
doi: 10.1038/s41396-018-0307-6
comments: true
---

## Abstract
Recent diversity studies have revealed that microbial communities of natural
environments are dominated by species-like, sequence-discrete populations.
However, how stable the sequence and gene-content diversity are within these
populations and especially in highly dynamic lotic habitats remain unclear.

Here we quantified the dynamics of intra-population diversity in samples
spanning two years and five sites in the Kalamas River (Northwest Greece).
A significant positive correlation was observed between higher intra-population
sequence diversity and longer persistence over time, revealing that more
diverse populations tended to represent more autochthonous (vs. allochthonous)
community members.
Assessment of intra-population gene-content changes caused by strain
replacement or gene loss over time revealed different profiles with the majority
of populations exhibiting gene-content changes close to 10% of the total genes,
while one population exhibited ~21% change.
The variable genes were enriched in hypothetical proteins and mobile elements,
and thus, were probably functionally neutral or attributable to phage predation.
A few notable exceptions to this pattern were also noted such as
phototrophy-related proteins in summer vs. winter populations.

Taken together, these results revealed that some freshwater genomes are
remarkably dynamic, even across short time and spatial scales, and have
implications for the bacterial species concept and microbial source tracking.
