---
layout: publication
title: "Freshwater Iterative Subtractive Binning"
date:   2020-06-14
excerpt: 'Iterative Subtractive Binning of Freshwater Chronoseries Metagenomes Identifies over 400 Novel Species and their Ecologic Preferences'
authors: 'Rodriguez-R LM, Tsementzi D, Luo C, Konstantinidis KT.'
citation: 'Env Micro (in press). 2020.'
category: Publication
tag:
- genomics
- metagenomics
- taxonomy
- microbial ecology
doi: 10.1111/1462-2920.15112
comments: true
---

## Summary
Recent advances in sequencing technology and bioinformatic pipelines have
allowed unprecedented access to the genomes of yet‐uncultivated microorganisms
from diverse environments.
However, the catalogue of freshwater genomes remains limited, and most genome
recovery attempts in freshwater ecosystems have only targeted specific taxa.
Here, we present a genome recovery pipeline incorporating iterative subtractive
binning, and apply it to a time series of 100 metagenomic datasets from seven
connected lakes and estuaries along the Chattahoochee River (Southeastern USA).
Our set of Metagenome‐Assembled Genomes (MAGs) represents >400 yet‐unnamed
genomospecies, substantially increasing the number of high‐quality MAGs from
freshwater lakes.
We propose names for two novel species: ‘*Candidatus* Elulimicrobium humile’
(‘*Ca.* Elulimicrobiota’, ‘Patescibacteria’) and ‘*Candidatus* Aquidulcis
frankliniae’ (‘Chloroflexi’).
Collectively, our MAGs represented about half of the total microbial community
at any sampling point.
To evaluate the prevalence of these genomospecies in the chronoseries, we
introduce methodologies to estimate relative abundance and habitat preference
that control for uneven genome quality and sample representation.
We demonstrate high degrees of habitat‐specialization and endemicity for most
genomospecies in the Chattahoochee lakes.
Wider ecological ranges characterized smaller genomes with higher coding
densities, indicating an overall advantage of smaller, more compact genomes for
cosmopolitan distributions.
