---
layout: publication
title:  "Using MiGA"
date:   2020-09-25
excerpt: "Classifying prokaryotic genomes using the Microbial Genomes Atlas (MiGA) webserver."
authors: 'Rodriguez-R LM, Harvey WT, Rosselló-Mora R, Tiedje JM, Cole JR, Konstantinidis KT.'
citation: 'Bergey's Manual of Systematics of Archaea and Bacteria. 2020.'
category: Publication
tag:
- genomics
- software
- taxonomy
- web app
doi: 10.1002/9781118960608.bm00042
software: miga
comments: true
---

## Abstract
The 16S rRNA gene has served effectively as the backbone of prokaryotic taxonomy
and phylogeny but offers limited resolution at the species level and is often
not assembled as part of metagenome‐assembled genomes (MAGs) recovered from
environmental DNA or single‐cell amplified genomes (SAGs). The genome‐aggregate
average nucleotide identity (ANI) and amino acid identity (AAI) have been shown
to provide robust resolution at the species and subspecies levels and can be
reliably calculated based on as little as ∼200 kb of a genome subsample. To
identify genomes of already described taxa as well as assess the degree of
novelty of genomes representing novel taxa and aid their taxonomic descriptions
based on the ANI/AAI concept, we have developed the Microbial Genomes Atlas
(MiGA) webserver ([www.microbial‐genomes.org](http://microbial‐genomes.org)).
Therefore, MiGA represents the genome equivalent of ribosomal webservers.
MiGA is regularly updated to include all available genomes of cultured and
uncultured (*Candidatus*) type material in its reference database and offers
additional bioinformatics services such as genome assembly from unassembled
reads, quality assessment, and detection of probable horizontal gene transfer
events from close relatives. In this article, we provide practical examples of
how to use MiGA and interpret its results. The advantages of MiGA over
alternative whole‐genome‐based approaches are also summarized.
